<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDetails</class>
 <widget class="QDialog" name="AccountDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #4a4a4a;
}

QDialog {
    background-color: #4a4a4a;
    color: #ffffff;
}

QPushButton {
background-color: gray;
border-radius: 10px;
color:lightblue;
}

QLabel#label_main_menu,
QLabel#label_transfers,
QLabel#label_withdrawal_deposit {
    font: 20pt &quot;Arial&quot;;
    color: lightyellow;
}

QLabel {
color:lightyellow;
font-family: Arial;
font-size: 15pt;
}

QLineEdit {
font-family: Arial;
font-size: 10pt;
background-color: lightgray;	
border: 1px solid white;
border-radius: 10px;
}

QTableView {
    background-color: #2e2e2e;
    color:lightblue;
border-radius: 10px;
}

QHeaderView::section {
    background-color: #1f1f1f;
	color:lightblue;
}

QTableView::item {
    border: 1px solid #3a3a3a;
}

QTableView::item:selected {
    background-color: #4a4a4a;
	color:lightblue;
 	gridline-color: gray
}
QHeaderView:vertical {
    background-color: #1f1f1f;
	color:lightblue;
    border: 1px solid #3a3a3a;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_delete_account">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>151</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Delete my account</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>551</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Account number</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_account_number">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Name holder</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_name">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txt_type">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <fontweight>Black</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 14pt &quot;Segoe UI Black&quot;;
color: lightgreen;</string>
     </property>
     <property name="text">
      <string>Account details</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>551</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>BALANCE:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_balance">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_change_password">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>250</y>
       <width>151</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Change Password</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="btn_delete_account_def">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>151</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Delete my account</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Give in your password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_password_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Arial&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Black</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <fontweight>Black</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 900 14pt &quot;Segoe UI Black&quot;;</string>
     </property>
     <property name="text">
      <string>Deleting is irreversible</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_go_back">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="btn_validate_password">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Validate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_go_back_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>341</width>
       <height>89</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Current password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_validate_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Give your new password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_new_password">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Arial&quot;;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Confirm your new password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_retry_new_password">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Arial&quot;;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_update_password">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>180</y>
       <width>131</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Update password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_go_back_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
